Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["macros", "xtask", "loco-extras"]
+members = ["macros", "xtask", "loco-extras", "loco-cli"]
```

Comment:
I wasn't entirely sure why loco-cli was not included in the main workspace, and added it so that rust-analyzer works when opening the repo.